src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2159,6 +2154,21 @@ private BoundExpression BindOutVariableArgument(DeclarationExpressionSyntax decl
                                         expressionVariableField, null, LookupResultKind.Viable, fieldType);
         }
 
+        /// <summary>
+        /// Returns true if a bad special by ref local was found.
+        /// </summary>
+        internal static bool CheckSpecialByRefLocal(Symbol containingSymbol, TypeSymbol type, DiagnosticBag diagnostics, SyntaxNode syntax)
```

Comment:
I'd rather not because all callers except one ignore the return value. The side effect is the main purpose.